src/charts/CytoViz/CytoViz.js

```diff
@@ -183,7 +328,12 @@ export const CytoViz = (props) => {
       />
       <div data-cy="cytoviz-open-drawer-button" className={classes.openDrawerButton}>
         <Tooltip TransitionComponent={Fade} TransitionProps={{ timeout: 600 }} title={labels_.settings.open}>
-          <IconButton onClick={openDrawer}>
+          <IconButton
+            onClick={() => {
+              openDrawer();
+              closeContextMenu();
```

Comment:
Is it really necessary to call `closeContextMenu();` ? The context menu should close automatically when users click outside of the menu